contracts/btc-finality/src/contract.rs

```diff
@@ -254,10 +260,12 @@ fn handle_end_block(
     Ok(res)
 }
 
-pub fn get_activated_height(staking_addr: &Addr, querier: &QuerierWrapper) -> StdResult<u64> {
+pub fn get_activated_height(querier: &QuerierWrapper) -> StdResult<u64> {
+    let params = get_babylon_sdk_params(querier)?;
```

Comment:
Nit: This is good for encapsulation, but bad for performance. In any case, since you already have it why not pass the entire `Config` by reference instead?